### DISCUSSION TITLE
Setting MAC, IP and LTSP # in dnsmasq.conf

### DISCUSSION THREAD

**Discussion by rweilert** [ltsp/ltsp#335] [Q&A]
When I edit dnsmasq.conf, the values set are not automatically read when restarting the system it seems. Or am I basically wrong there when identifying MAC against IP and LTSP number?

I have a dnsmasq.conf which gives the MACs and the IPs and LTSP numbers for every client. Recently I had to replace one client so that number 141 should become 104. I edited the file, but even after reboot the changings haven't come into effect.

What am I missing? 

Regards
Rolf

**Comment by alkisg** [comment]
If you're using dnsmasq in proxy DHCP mode, which is the default for LTSP, then it doesn't give IPs to clients. Your real DHCP server, e.g. your router, gives them.

What are you trying to do, give specific IPs and HOSTNAMEs to LTSP clients? You can use ltsp.conf for that.

**Comment by rweilert** [comment]
No, I don't have proxy DHCP mode, because it is not my router which gives the IPs. There are 2 NICs, one for the router only (outbound) and one for LTSP and all internal network (printers etc.) with 192.168.67.x. The server is the DHCP instance.

There is a long list with all the clients in dnsmasq.conf, so I thought this list be in use. Just can't remember what I did when I built the server, it's been too long ago and I have never had to touch it again :) (Good software, though :) )

So you say ltsp.conf - but where is it?

Rolf

**Comment by alkisg** [comment]
If you're using dnsmasq in real mode, i.e. if your clients have 192.168.67.x IPs, then sure your dnsmasq.conf should work fine. Try to paste it here or in the dnsmasq mailing list so that someone can spot any issues.

Regarding ltsp.conf, run `man ltsp.conf` to learn where it is etc.

**Comment by alkisg** [comment]
Ah I just noticed in your other question that you're using LTSP 5. This github repository and discussions are only for the new LTSP, to avoid confusing the users with conflicting answers.

**Comment by rweilert** [comment]
Ok, where then?

**Comment by alkisg** [comment]
Go to ltsp.org and search for LTSP5, there are link there for all its resources.

But it sounds like your dnsmasq is fine, and you just have an "epoptes alias" that's wrong. If you open a terminal on the client, does it show the correct hostname?
